angular-ngrx-scss/src/app/file-viewer/file-explorer-root/file-explorer-root.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { ActivatedRoute } from '@angular/router';
+import { fetchRepository, selectRepositoryState } from '../../state/repository';
+
+@Component({
+  selector: 'app-file-explorer-root',
+  templateUrl: './file-explorer-root.component.html',
+  styleUrls: ['./file-explorer-root.component.scss'],
+})
+export class FileExplorerRootComponent implements OnInit {
+  repo$ = this.store.select(selectRepositoryState);
+
+  constructor(private route: ActivatedRoute, private store: Store) {}
+
+  ngOnInit() {
+    const owner = this.route.snapshot.paramMap.get('owner') as string;
```

Comment:
suggestion: We should be able to have this data already in the store. Can we not pull it from there, rather than grabbing values from the route?

- [ ] Use store to get owner and repo name instead of router

Reply:
I think I may be a bit confused here. Can we look through this via a pair session?